docs/ui/cloud-getting-started.md

```diff
@@ -16,11 +16,11 @@ tags:
 The following sections will get you set up and using Prefect Cloud, using these steps:
 
 1. [Sign in or register](#sign-in-or-register) a Prefect Cloud account.
-2. [Create workspaces](#create-a-workspace) for your account.
-3. [Create an API key](#create-an-api-key) to authorize a local execution environment.
-4. [Configure Orion settings](#configure-execution-environment) to use Prefect Cloud.
-5. [Configure storage](#configure-storage).
-6. [Run a flow](#run-a-flow-with-prefect-cloud) and view flow results in Prefect Cloud.
+1. [Create workspaces](#create-a-workspace) for your account.
+1. [Create an API key](#create-an-api-key) to authorize a local execution environment.
+1. [Configure a local execution environment](#configure-a-local-execution-environment) to use Prefect Cloud.
+1. [Run a flow](#run-a-flow-with-prefect-cloud) locally and view flow results in Prefect Cloud.
```

Comment:
```suggestion
1. [Run a flow](#run-a-flow-with-prefect-cloud) locally and view flow run execution in Prefect Cloud.
```

otherwise users could confuse it with being able to see run artifacts there